api/src/main/java/io/strimzi/api/kafka/model/template/ContainerTemplate.java

```diff
@@ -41,6 +43,16 @@ public void setEnv(List<ContainerEnvVar> env) {
         this.env = env;
     }
 
+    @Description("Security context for the container")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
I guess it has some Kubernetes reference .... can we add the `@KubeLink` annotation to render it in the doc? It is done similarly in the `PodSecurityContext` in `PodTemplate`.